server/src/main/java/keywhiz/ServiceModule.java

```diff
@@ -181,12 +185,16 @@ public ServiceModule(KeywhizConfig config, Environment environment) {
   }
 
   @Provides @Singleton
-  @Readonly DBI readonlyDbi(DBIFactory factory, Environment environment, KeywhizConfig config,
-      MapArgumentFactory mapArgumentFactory, @Readonly ManagedDataSource dataSource)
+  @Readonly DBI readonlyDbi(KeywhizConfig config, MapArgumentFactory mapArgumentFactory,
```

Comment:
I have another cleanup PR after this one. You can diff my branch vs this
branch if you want to see what all will go away.

On Tue, Apr 21, 2015 at 6:23 PM, Justin Cummins <notifications@github.com>
wrote:

> In server/src/main/java/keywhiz/ServiceModule.java
> <https://github.com/square/keywhiz/pull/52#discussion_r28837861>:
>
> > @@ -181,12 +185,16 @@ public ServiceModule(KeywhizConfig config, Environment environment) {
> >    }
> >
> >    @Provides @Singleton
> > -  @Readonly DBI readonlyDbi(DBIFactory factory, Environment environment, KeywhizConfig config,
> > -      MapArgumentFactory mapArgumentFactory, @Readonly ManagedDataSource dataSource)
> > +  @Readonly DBI readonlyDbi(KeywhizConfig config, MapArgumentFactory mapArgumentFactory,
>
> DBI still needed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/square/keywhiz/pull/52/files#r28837861>.
>